src/main/java/org/opencds/cqf/cdshooks/request/JsonFieldResolution.java

```diff
@@ -64,7 +64,7 @@ public static JsonArray getArrayField(JsonObject object, String field, boolean r
     }
 
     public static Boolean validateNull(JsonElement field, String fieldName, boolean required) {
-        if (field == null) {
+        if (field == null || field.isJsonNull()) {
```

Comment:
This method is simply checking if a required field exists. If the field is specified as a JSON null value, then it must exist. If the field is a JSON null, then it will throw a more appropriate InvalidFieldTypeException in the calling functions. This method should be private however.

Reply:
I'll make that change in my next commit.